ct/wallos.sh

```diff
@@ -63,14 +63,19 @@ if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_v
   msg_info "Updating ${APP} to ${RELEASE}"
   cd /opt
   wget -q "https://github.com/ellite/Wallos/archive/refs/tags/v${RELEASE}.zip"
-  mv /opt/wallos/db/wallos.db /opt/wallos.db
-  mv /opt/wallos/images/uploads /opt/images/uploads
+  mkdir -p /opt/uploads_backup
+  if [ -f "/opt/wallos/db/wallos.db" ]; then
+  mv "/opt/wallos/db/wallos.db" "/opt/wallos.db"
+  fi
+  mv /opt/wallos/images/uploads/* /opt/uploads_backup/
   unzip -q v${RELEASE}.zip
   rm -rf /opt/wallos
   mv Wallos-${RELEASE} /opt/wallos
   rm -rf /opt/wallos/db/wallos.empty.db
-  mv /opt/wallos.db /opt/wallos/db/wallos.db
-  mv /opt/images/uploads /opt/wallos/images/uploads
+  if [ -f "/opt/wallos.db" ]; then
+  mv "/opt/wallos.db" "/opt/wallos/db/wallos.db"
+  fi
+  mv /opt/uploads_backup/* /opt/wallos/images/uploads
```

Comment:
`mv: cannot move '/opt/uploads_backup/icons' to '/opt/wallos/images/uploads/icons': Directory not empty`
It's a bit ugly by the original author, but uploads/icons contain the default icons :(

Reply:
Then only we delete the folder before

Reply:
I did a code search in https://github.com/ellite/Wallos, an if I understand correctly ONLY /opt/wallos/images/uploads/logos/ is used for user-uploaded images. (which is probably also why only this directory and the db directory are mentioned as docker volumes in the [docker instructions](https://github.com/ellite/Wallos?tab=readme-ov-file#docker-1))

Reply:
so switched to logo. Can you Test?